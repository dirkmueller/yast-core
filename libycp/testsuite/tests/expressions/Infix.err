Parsed:
----------------------------------------------------------------------
(1 + 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 - 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 * 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 / 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 % 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 << 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 >> 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 & 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 | 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
~ 1
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true && false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(true || false)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 == 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 < 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 > 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 <= 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 >= 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 != 1)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
false
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
0
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
1
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
(1 > 0) ? 1 : 0
----------------------------------------------------------------------
[Parser] tests/expressions/Infix.ycp:33 Expression before '?' must be boolean
[Parser] tests/expressions/Infix.ycp:33 Expected 'boolean', seen 'integer'.
runycp: parser error
