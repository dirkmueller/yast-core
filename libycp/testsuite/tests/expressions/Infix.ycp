// infix expressions

(1+1)
(1-1)
(1*1)
(1/1)
(1%1)
(1<<1)
(1>>1)
(1&1)
(1|1)

(~1)

(true && false)
(true || false)


(1 == 1)
(1 < 1)
(1 > 1)
(1 <= 1)
(1 >= 1)
(1 != 1)

(!true)

(false?1:0)
(true?1:"a")
((1>0)?1:0)
(
// Line 33
(1+0)
?
1
:
0
)
(
[true]
?"a":1)
(
$[1:true]
?"a":1)
(-1)
