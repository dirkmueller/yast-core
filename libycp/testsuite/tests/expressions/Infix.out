(2)
(0)
(1)
(1)
(0)
(2)
(0)
(1)
(1)
(-2)
(false)
(true)
(true)
(false)
(false)
(true)
(true)
(false)
(false)
(0)
(1)
(1)
